index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <main>
+
+        <nav id="nav-bar">
+            <div class="left-nav-bar">
+                <img class="menu-logo" src="https://img.icons8.com/material-rounded/24/000000/menu.png" />
+                <img class="YouTube-logo" src="https://img.icons8.com/fluent/48/000000/youtube-play.png" />
+                <h3 class="title">YouTube</h3>
+            </div>
+            <div class="mid-nav-bar">
+                <input type="text">
+                <button>Search</button>
+            </div>
+            <div class="right-nav-bar">
+                <img class="icon video" src="https://img.icons8.com/metro/52/000000/video-call.png" />
+                <img class="icon bell" src="https://img.icons8.com/android/24/000000/appointment-reminders.png" />
+                <img class="icon avatar"
+                    src="https://img.icons8.com/dusk/64/000000/circled-user-female-skin-type-3.png" />
+
+
+            </div>
+
+
+
+        </nav>
+        <section>
+            <div class="video-player">
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/YaioUnMw0mo" frameborder="0"
+                    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                    allowfullscreen></iframe>
+                <h2 id="video-tite">React App - Breaking Bad API
+                </h2>
+                <p class="views"><strong>69,941 views</strong> </p>
+
+                <img class="rate like" src="img/like-button.jpg" alt="">
+                <label class="like-text">32.1k</label>
+                <img class="rate dislike" src="img/dislike.jpg" alt="">
+                <label class="share-text">share</label>
```

Comment:
You have a share, However, this should a button like specified in the requirements :arrow_down: 
![video_share_requirement](https://user-images.githubusercontent.com/32361963/87135508-cb542000-c2a2-11ea-8279-423948b61162.png)